packages/xrpl/test/wallet/index.test.ts

```diff
@@ -56,7 +56,13 @@ describe('Wallet', function () {
       assert.isTrue(wallet.privateKey.startsWith(ed25519KeyPrefix))
       assert.isTrue(wallet.classicAddress.startsWith(classicAddressPrefix))
     })
-
+    
+    it('generates a new wallet using an invalid/unknown algorithm', function () {
+      const algorithm = "test"
+      
+      assert.throws(() => { Wallet.generate(algorithm) }, /Invalid cryptographic signing algorithm/)
```

Comment:
Need to add `//@ts-expect-error`.  This will allow the test to compile even though the type is wrong to simulate how it might work if using js.

Reply:
Or just cast it via `as unknown as ExpectedType`